monai/transforms/spatial/array.py

```diff
@@ -1085,6 +1255,9 @@ class AffineGrid(Transform):
             pixel/voxel relative to the center of the input image. Defaults to no translation.
         scale_params: scale factor for every spatial dims. a tuple of 2 floats for 2D,
             a tuple of 3 floats for 3D. Defaults to `1.0`.
+        dtype: data type for the grid computation. Defaults to ``np.float32``.
```

Comment:
I want to double confirm here the `grid computation` is with `float32`, other `resampling computation` is `float64`?

Thanks.

Reply:
yes I think it's good, these are also non-breaking changes